django_mongodb_backend/management/commands/get_encrypted_fields_map.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections, router
+
+
+class Command(BaseCommand):
+    help = "Generate a `schema_map` of encrypted fields for all encrypted"
+    " models in the database for use with `get_autoencryption_opts` in"
+    " production environments."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--database",
+            default=DEFAULT_DB_ALIAS,
+            help="Specify the database to use for generating the encrypted"
+            "fields map. Defaults to the 'default' database.",
+        )
+
+    def handle(self, *args, **options):
+        db = options["database"]
+        connection = connections[db]
+
+        schema_map = self.generate_encrypted_fields_schema_map(connection)
+
+        self.stdout.write(json.dumps(schema_map, indent=2))
+
+    def generate_encrypted_fields_schema_map(self, conn):
+        schema_map = {}
+
+        for app_config in apps.get_app_configs():
+            for model in router.get_migratable_models(
+                app_config, conn.alias, include_auto_created=False
+            ):
+                if getattr(model, "encrypted", False):
+                    encrypted_fields = self.get_encrypted_fields(model, conn)
```

Comment:
You can still initialize `with conn.schema_editor() as editor:` outside the loops and change this line to  `editor._get_encrypted_fields_map(model)`.

Reply:
2386397fcc6a4861ca7805004bbf114863b6fdc0